src/App.test.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import App from './App';
+
+it('renders without crashing', () => {
+    const div = document.createElement('div');
+    ReactDOM.render(<App />, div);
```

Comment:
does this actually pass a true/false test? I mean can it fail? you could do a shallow render of App and then test to see if the class is there. expect(wrapper.hasClass(“App")).to.equal(true);